packages/web/src/scss/components/Drawer/_Drawer.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../tools/typography';
+@use '@tokens' as tokens;
+@use 'theme';
+
+.Drawer {
+    @include typography.generate(theme.$drawer-typography);
+
+    --#{tokens.$css-variable-prefix}drawer-translate-x: 0%;
+
+    all: unset;
```

Comment:
(@adamkudrna) maybe it would be nice to add a reset mixin for this. I assume it is partly the same as in Modal (and HeaderDialog)

Reply:
You mean for this line? 👀  Or something like `reset.modal()`?